packages/design/src/LabelState/LabelState.test.tsx

```diff
@@ -47,12 +48,12 @@ describe('design/LabelState', () => {
       background: expected,
     });
 
-    expect(getComputedStyle(container.firstChild).boxShadow).toBe('');
+    expect(getComputedStyle(container.firstElementChild).boxShadow).toBe('');
```

Comment:
[nit]
```ts
expect(container.firstElementChild).not.toHaveStyleRole('boxShadow');
```

Reply:
tried this, but you have to provide a full CSS value (`box-shadow: whatever`) 

The value of this is computed, so I cant easily use `toHaveStyle` (I can't find any info about `toHaveStyleRole` anywhere) - I'm just going to leave it as it is